js/ui/handler/drag_rotate.js

```diff
@@ -17,9 +17,10 @@ var inertiaLinearity = 0.25,
  * dragging the cursor while holding the right mouse button or the `ctrl` key.
  * @class DragRotateHandler
  */
-function DragRotateHandler(map) {
+function DragRotateHandler(map, options) {
     this._map = map;
     this._el = map.getCanvasContainer();
+    this._options = options;
```

Comment:
Better to explicitly save `this._bearingSnap` here instead of the options object.